src/style.css

```diff
@@ -1,3 +1,80 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --header-height: 60px;
+  --primary-color: #00acdc;
+}
+
+.header {
+  background-color: #fff;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)